### PR TITLE
fix: #1276

### LINKED ISSUE #1276: Blog should not be accessible on Filecoin deployment
We need to start redirecting `/blog/` routes to the main deployment if in "alternative chain mode".

### PR DESCRIPTION
**Pull request by efstajas**
Minor visual tweaks for network picker.

- Fixes unstyled text color for network picker in mobile account bottom sheet
- Fixes width of border outline for currently selected network
- Fixes color of checkmark icon

Resolves #1276

### CODE REVIEW

File: src/lib/components/icons/IconWrapper.svelte [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 <!-- svelte-ignore a11y-no-static-element-interactions -->
 <svg
   on:click
   on:keydown
-  style={`flex-shrink: 0; ${style ? style : ''}`}
   width="24"
   height="24"
-  fill="var(--color-foreground-level-5)"
+  fill={fill ?? 'var(--color-foreground-level-5)'}
+  style={`flex-shrink: 0; ${style ? style : ''}`}
```
Comment (mhgbrown) [inline src/lib/components/icons/IconWrapper.svelte:13]: Is this order enforced via formatting?

**Review by mhgbrown — APPROVED** [review]
Looking goody, just left a question!

File: src/lib/components/icons/IconWrapper.svelte (hunk 1 above)
Comment (efstajas) [reply src/lib/components/icons/IconWrapper.svelte:13]: No, that change is actually just unnecessary^^ not sure why I did that

**Comment by efstajas** [comment]
Woops! This is actually about issue #1275 😇

**Pull request merged**